visallo/visallo
# fix check for valid coordinates

Testing Instructions:
- Create a entity with geolocation latitude: `24.5333` & longitude: `111.5167`.
- Drag and drop entity onto a map work product
- entity should show up on map

Points of Regression: N/A

NO CHANGELOG
